x-pack/plugins/apm/public/components/app/settings/agent_configurations/agent_configuration_create_edit/settings_page/setting_form_row.tsx

```diff
@@ -163,7 +164,7 @@ export function SettingFormRow({
       }
       description={
         <>
-          {setting.description}
+          <ReactMarkdown>{setting.description}</ReactMarkdown>
```

Comment:
FYI: `Eui` has a [formatter](https://elastic.github.io/eui/#/editors-syntax/markdown-format#built-in-plugins) that render the markdown and it seems to be used

Reply:
ohhh! I liked that one better.
<img width="988" alt="image" src="https://user-images.githubusercontent.com/1313018/191280340-2d5ef599-86ed-45c0-8c18-9d69c1997dc4.png">

should we use that one instead?

Reply:
it looks nice 🤩  !

`EuiMarkdownFormat` is labeled as beta but it was released on [28.0.0](https://github.com/elastic/eui/tree/v28.0.0)  - more than 2 years ago, so I think we won't have any issue.

Reply:
https://github.com/elastic/eui/pull/3522